EDU-483: Add Data Explorer bucket removal (#426)
seqeralabs/docs: platform_versioned_docs/version-24.2/data/data-explorer.mdx

  Select **Add cloud bucket** from the Data Explorer tab to add individual buckets (or directory paths within buckets). Specify the **Provider**, **Bucket path**, **Name**, **Credentials**, and **Description**, then select **Add**. For public cloud buckets, select **Public** from the **Credentials** drop-down menu.

## Remove cloud storage buckets

A workspace maintainer can remove a manually created cloud storage bucket.

From the **Data Explorer** tab, find the cloud bucket that you want to delete. Select the options menu for the bucket, and select **Remove**. When prompted, select **Remove** from the confirmation modal that appears.

> If you remove a cloud storage bucket that is associated as a data link with a Studio, the data link is automatically removed from the Studio configuration.

Reviewer: I find this very misleading - the user can remove the data link in platform, but _not the actual cloud bucket_. This seems to suggest otherwise

> If you remove a cloud storage bucket that is associated as a data link with a Studio, the data link is automatically removed from the Studio configuration.

Reviewer 2: ```suggestion
If you remove a data link associated with a cloud storage bucket, the data link is automatically removed from the relevant Studio configuration.
```

Reviewer 2: @alvaromartmart, does this work better?
